api/featuresserver.go

```diff
@@ -90,26 +100,38 @@ type LoadControlServerInterface interface {
 	) *model.LoadControlLimitIdType
 
 	// Set or update data set for a limitId
-	// Elements provided in deleteElements will be removed from the data set before the update
+	// Id provided in deleteId will trigger removal of matching items from the data set before the update
+	// Elements provided in deleteElement will limit the fields to be removed using Id
 	//
 	// Will return an error if the data set could not be updated
-	UpdateLimitDataForId(
-		data model.LoadControlLimitDataType,
+	UpdateLimitDataForIds(
+		data []LoadControlLimitDataForID,
+		deleteId *model.LoadControlLimitIdType,
```

Comment:
Why the Id is called `deleteId` when u have a method to update the limit?
It feels like the method name and the id doesn't match to each other

Reply:
On each update, you can also request to delete some data. So the `data` structure contains everything you want to set, and `deleteId` and `deleteElements` references what you want to delete.

Reply:
But shouldn't that be an delete instead of an update method?
It's confusing why I need to pass a delete ID for an CRUD method named update..

Reply:
It's to allow deletion of a field item, e.g. duration in LPC. The `data` structure doesn't allow you that. Having separate methods would requiring to introduce transactions, which I want to avoid.

The reasoning behind all that is how EEBUS works. See specs about partial updates.

Reply:
One addition: the complete updates are always partial and only one delete element can be done on a single SPINE message. That's why the `deleteId` and `deleteElements` are not part of the `data` structure.